<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuPrincipal</class>
 <widget class="QMainWindow" name="MenuPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1375</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1375</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Monitoreo de tortugas v1.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/Images/if_Turtle_204384.ico</normaloff>:/Images/if_Turtle_204384.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(57, 134, 137)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>1240</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>841</width>
      <height>490</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1240</width>
      <height>600</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Raleway</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(206, 206, 206);
background-color: rgb(255, 170, 255);
background-color: rgb(231, 231, 231);</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tabWelcome">
     <attribute name="title">
      <string>Bienvenida</string>
     </attribute>
     <widget class="QLabel" name="lbTittle">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>110</y>
        <width>721</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>¡Turtles finder te da la bienvenida!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lbMessage">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>220</y>
        <width>1151</width>
        <height>271</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway Medium</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Este proyecto fue hecho y mayormente pensado para ayudar a una de las especies en mayor
peligro de extinción:  Las tortugas marinas.

Nuestro propósito es llevar a cabo un rastreo de las tortugas para poder conocer a mayor
profundidad cuales son sus zonas de alimentación, su ruta de migración, rutas de
caza de las tortugas, etc.

En la parte de abajo se colocaran hipervínculos de ayuda para mejorar la experencia del
usuario dentro del sistema.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>100</y>
        <width>91</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/Images/if_Turtle_204384.ico</normaloff>:/Images/if_Turtle_204384.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="lbBack">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1241</width>
        <height>571</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>lbBack</zorder>
     <zorder>lbTittle</zorder>
     <zorder>lbMessage</zorder>
     <zorder>pushButton</zorder>
    </widget>
    <widget class="QWidget" name="tabAdmistratation">
     <attribute name="title">
      <string>Administración</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>140</y>
        <width>400</width>
        <height>341</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>371</width>
        <height>341</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>341</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="pbTurtles">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>300</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Raleway</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"> QPushButton#pbTurtles {
background-color: rgb(120, 190, 151);
 }

 QPushButton#pbTurtles:hover {
background-color: rgb(56, 134, 136);
color:  rgb(255, 255, 255);
 }</string>
         </property>
         <property name="text">
          <string>Administrar tortugas</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/Images/if_Turtle_204384.ico</normaloff>:/Images/if_Turtle_204384.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pbState">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>300</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Raleway</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"> QPushButton#pbState {
background-color: rgb(120, 190, 151);
 }

 QPushButton#pbState:hover {
background-color: rgb(56, 134, 136);
color:  rgb(255, 255, 255);
 }</string>
         </property>
         <property name="text">
          <string> Agregar estados y municipios</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/Images/world.ico</normaloff>:/Images/world.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pbSub">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>300</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Raleway</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"> QPushButton#pbSub {
background-color: rgb(120, 190, 151);
 }

 QPushButton#pbSub:hover {
background-color: rgb(56, 134, 136);
color:  rgb(255, 255, 255);
 }</string>
         </property>
         <property name="text">
          <string>Administrar subespecies</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/Images/if_Turtle_204384.ico</normaloff>:/Images/if_Turtle_204384.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pbZone">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>300</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Raleway</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"> QPushButton#pbZone {
background-color: rgb(120, 190, 151);
 }

 QPushButton#pbZone:hover {
background-color: rgb(56, 134, 136);
color:  rgb(255, 255, 255);
 }</string>
         </property>
         <property name="text">
          <string>   Administrar zonas</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/Images/379538.ico</normaloff>:/Images/379538.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="lbGif">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>140</y>
        <width>450</width>
        <height>331</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>371</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>450</width>
        <height>370</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lbTitleAdmin">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>50</y>
        <width>461</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>17</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Administración de recursos</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pbTurtle">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>91</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/Images/if_Turtle_204384.ico</normaloff>:/Images/if_Turtle_204384.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabMap">
     <attribute name="title">
      <string>Mapa</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>160</y>
        <width>376</width>
        <height>330</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>371</width>
        <height>330</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>330</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QPushButton" name="pbSensor">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>75</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>75</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Raleway</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"> QPushButton#pbSensor {
background-color: rgb(120, 190, 151);
 }

 QPushButton#pbSensor:hover {
background-color: rgb(56, 134, 136);
color:  rgb(255, 255, 255);
 }</string>
         </property>
         <property name="text">
          <string>Sensor de tortugas</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/Images/381031-200.ico</normaloff>:/Images/381031-200.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="psbMapa">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>75</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>75</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Raleway</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"> QPushButton#psbMapa {
background-color: rgb(120, 190, 151);
 }

 QPushButton#psbMapa:hover {
background-color: rgb(56, 134, 136);
color:  rgb(255, 255, 255);
 }</string>
         </property>
         <property name="text">
          <string>  Mapa de ubicaciones</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/Images/map-map-marker-icon_34394.ico</normaloff>:/Images/map-map-marker-icon_34394.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="psbExtrapolacion">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>75</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>75</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Raleway</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"> QPushButton#psbExtrapolacion {
background-color: rgb(120, 190, 151);
 }

 QPushButton#psbExtrapolacion:hover {
background-color: rgb(56, 134, 136);
color:  rgb(255, 255, 255);
 }</string>
         </property>
         <property name="text">
          <string>Extrapolación de avance</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/Images/Turtli.png</normaloff>:/Images/Turtli.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="lbGif_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>180</y>
        <width>440</width>
        <height>300</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>440</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>440</width>
        <height>300</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lbTitleMap">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>60</y>
        <width>611</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>17</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Manejo de sensores y visualización de mapa</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pbTurtle2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>40</y>
        <width>91</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/Images/if_Turtle_204384.ico</normaloff>:/Images/if_Turtle_204384.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Estadísticas y reportes</string>
     </attribute>
     <widget class="QPushButton" name="pbStadistics">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>180</y>
        <width>350</width>
        <height>75</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>75</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>75</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"> QPushButton#pbStadistics {
background-color: rgb(120, 190, 151);
 }

 QPushButton#pbStadistics:hover {
background-color: rgb(56, 134, 136);
color:  rgb(255, 255, 255);
 }</string>
      </property>
      <property name="text">
       <string>Estadísticas de la tortuga</string>
      </property>
      <property name="icon">
       <iconset resource="images2.qrc">
        <normaloff>:/stats_icon-icons.com_60590_60590.ico</normaloff>:/stats_icon-icons.com_60590_60590.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pbReport">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>280</y>
        <width>350</width>
        <height>75</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>75</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>75</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"> QPushButton#pbReport {
background-color: rgb(120, 190, 151);
 }

 QPushButton#pbReport:hover {
background-color: rgb(56, 134, 136);
color:  rgb(255, 255, 255);
 }</string>
      </property>
      <property name="text">
       <string>   Reporte de la tortuga</string>
      </property>
      <property name="icon">
       <iconset resource="images2.qrc">
        <normaloff>:/report.ico</normaloff>:/report.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pbTurtle2_3">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>91</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/Images/if_Turtle_204384.ico</normaloff>:/Images/if_Turtle_204384.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="lbTitleMap_3">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>60</y>
        <width>611</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>17</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Estadísticas y reportes de las tortugas</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lbGif_5">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>160</y>
        <width>440</width>
        <height>300</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>440</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>440</width>
        <height>300</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pbReportView">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>380</y>
        <width>350</width>
        <height>75</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>75</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>75</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"> QPushButton#pbReportView {
background-color: rgb(120, 190, 151);
 }

 QPushButton#pbReportView:hover {
background-color: rgb(56, 134, 136);
color:  rgb(255, 255, 255);
 }</string>
      </property>
      <property name="text">
       <string>  Ver mis reportes</string>
      </property>
      <property name="icon">
       <iconset resource="images2.qrc">
        <normaloff>:/report_edit.ico</normaloff>:/report_edit.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
  <include location="images2.qrc"/>
 </resources>
 <connections/>
</ui>
